src/TipBox.vue

```diff
@@ -105,19 +114,19 @@
       lightStyle() {
         if (this.light) {
             switch (this.type) {
-            case 'warning':
-                return 'border-warning text-warning alert-border-left';
-            case 'info':
-            case 'definition':
-                return 'border-info text-info alert-border-left';
-            case 'success':
-            case 'tip':
-                return 'border-sucess text-success alert-border-left';
-            case 'important':
-            case 'wrong':
-                return 'border-danger text-danger alert-border-left';
-            default:
-                return '';
+              case 'warning':
+                  return 'border-warning text-warning alert-border-left';
+              case 'info':
+              case 'definition':
+                  return 'border-info text-info alert-border-left';
+              case 'success':
+              case 'tip':
+                  return 'border-sucess text-success alert-border-left';
+              case 'important':
+              case 'wrong':
+                  return 'border-danger text-danger alert-border-left';
+              default:
+                  return '';
```

Comment:
Avoid unnecessary code style changes?

Reply:
The indentation was wrong.

Reply:
Code style changes should be made in another PR

Reply:
Ok. Noted!